db/schema.rb

```diff
@@ -175,10 +175,26 @@
     t.uuid "school_id", null: false
     t.datetime "created_at", precision: 6, null: false
     t.datetime "updated_at", precision: 6, null: false
+    t.uuid "partnership_notification_email_id"
+    t.index ["partnership_notification_email_id"], name: "index_nomination_emails_on_partnership_notification_email_id"
     t.index ["school_id"], name: "index_nomination_emails_on_school_id"
     t.index ["token"], name: "index_nomination_emails_on_token", unique: true
   end
 
+  create_table "partnership_notification_emails", id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
+    t.string "token", null: false
+    t.string "sent_to", null: false
+    t.string "email_type", null: false
+    t.string "notify_id"
```

Comment:
I think you need to index this column as the callback from notify will lookup using the notify id